src/test/java/hudson/plugins/active_directory/ActiveDirectoryJCasCCompatibilityTest.java

```diff
@@ -66,4 +62,19 @@ protected void assertConfiguredAsExpected(RestartableJenkinsRule restartableJenk
     protected String stringInLogExpected() {
         return "Setting class hudson.plugins.active_directory.ActiveDirectorySecurityRealm.groupLookupStrategy = RECURSIVE";
     }
+
+    @Extension
```

Comment:
For unknown reason `@TestExtension` didn't work. I am guessing because of the inheritance and `RestartableJenkinsRule` perhaps?